great_expectations/expectations/metrics/map_metric_provider/map_condition_auxilliary_methods.py

```diff
@@ -435,15 +433,15 @@ def _sqlalchemy_map_condition_query(
         unexpected_condition,
         compute_domain_kwargs,
         accessor_domain_kwargs,
-    ) = metrics.get("unexpected_condition")
+    ) = metrics["unexpected_condition"]
 
     result_format: dict = metric_value_kwargs["result_format"]
     # We will not return map_condition_query if return_unexpected_index_query = False
-    return_unexpected_index_query: bool = result_format.get(
+    return_unexpected_index_query: Optional[bool] = result_format.get(
         "return_unexpected_index_query"
     )
     if return_unexpected_index_query is False:
-        return
+        return None
```

Comment:
This logic is weird to me - we exit early only if we have `False`? What if the value is not in the dict? I decided to just update the type annotation but could this be a bug?

Reply:
Maybe the default is to return the value so this makes sense